### PR TITLE
Set "debug" through external properties.

### PR DESCRIPTION
**Pull request by maciejwalkowiak**
## :scroll: Description

Allows setting "debug" through external properties.

## :bulb: Motivation and Context

Useful when debugging Spring (non-boot) integration.

## :green_heart: How did you test it?

Unit tests.

## :pencil: Checklist

- [x] I reviewed submitted code
- [x] I added tests to verify changes
- [x] All tests passing
- [x] No breaking changes

### CODE REVIEW

File: sentry/src/main/java/io/sentry/SentryOptions.java [hunk 1]
```diff
@@ -353,18 +354,27 @@ public void setDsn(@Nullable String dsn) {
...
+  public void setDebug(final @Nullable Boolean debug) {
     this.debug = debug;
   }
 
+  /**
+   * Check if debug mode is ON, OFF or not set.
+   *
+   * @return true if ON or false otherwise
+   */
+  public @Nullable Boolean getDebug() {
```
Comment (marandaneto) [inline sentry/src/main/java/io/sentry/SentryOptions.java:374]: can we make it either private or package-private? `isDebug()` is all we need.

Comment (maciejwalkowiak) [reply sentry/src/main/java/io/sentry/SentryOptions.java]: Package private 👍

Comment (maciejwalkowiak) [reply sentry/src/main/java/io/sentry/SentryOptions.java]: .. or actually private would be better.

**Review by bruno-garcia — COMMENTED** [review]
Couldn't we get away with `boolean` and default to `false`? You only need to set the value if it's `true` anyway (since we always have it false)

**Comment by maciejwalkowiak** [comment]
@bruno-garcia practically yes, but theoretically there can be a case when it is set to `true` in the code and then turned off in external configuration.

**Review by bruno-garcia — APPROVED** [review]

**Review by marandaneto — APPROVED** [review]

**Pull request merged**